bittensor/_neuron/text/template_miner/neuron_impl.py

```diff
@@ -69,13 +68,16 @@ def __init__( self, config: 'bittensor.config', nucleus: 'Nucleus'):
         self.nucleus.metagraph = self.metagraph_callback
         self.nucleus.dendrite = self.dendrite
         self.optimizer = torch.optim.SGD(
-            [ {'params': self.nucleus.peer_weights, 'lr': self.config.neuron.learning_rate_chain} ],
```

Comment:
if the learning_rate_chain is not needed here, might as well remove it from the config